instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -30,21 +30,27 @@ def _extract_conn_attributes(conn_kwargs):
     }
     db = conn_kwargs.get("db", 0)
     attributes[SpanAttributes.DB_REDIS_DATABASE_INDEX] = db
-    try:
+
+    if conn_kwargs.get("username"):
+        attributes[SpanAttributes.DB_USER] = conn_kwargs["username"]
+
+    if conn_kwargs.get("client_name"):
+        # TODO: add to semconv?
+        attributes["db.redis.client_name"] = conn_kwargs["client_name"]
```

Comment:
This isn't in the semconv, but would be useful for debugging and can be different from `username`.